Backend/Remora.Discord.API.Abstractions/API/Objects/Interactions/ApplicationCommands/IApplicationCommand.cs

```diff
@@ -104,4 +100,14 @@ public interface IApplicationCommand
     /// This field is only supplied by Discord as a response, and is not used to set the actual localized string.
     /// </remarks>
     Optional<string> DescriptionLocalized { get; }
+
+    /// <summary>
+    /// Gets a value that indicates the requisite permissions to execute the command.
+    /// </summary>
+    Optional<IDiscordPermissionSet?> DefaultMemberPermissions { get; }
+
+    /// <summary>
+    /// Gets a value that indicates whether this command can be executed in DMs.
+    /// </summary>
+    Optional<bool> DMPermission { get; }
```

Comment:
```suggestion
    Optional<bool?> DMPermission { get; }
```

Should be nullable according to the spec.